### PR TITLE
feat(app): Added skeleton for solana app proto decoding

### PR DESCRIPTION
**Pull request by amanCypherock**
https://app.clickup.com/t/9002019994/PRF-4963

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-4963 Solana application skeleton](https://app.clickup.com/t/9002019994/PRF-4963)

File: src/menu/host_interface.c [hunk 1]
```diff
@@ -149,6 +150,10 @@ void main_menu_host_interface(engine_ctx_t *ctx,
       near_main(usb_evt);
       break;
     }
+    case 9: {
```
Comment (kartikCypherock) [inline src/menu/host_interface.c:153]: Move app id for solana to 9 (Coin_Type + 1, as 1 is given to manager app)

Comment (amanCypherock) [reply src/menu/host_interface.c]: updated

**Review by kartikCypherock — APPROVED** [review]
LGTM

**Pull request merged**